Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -357,6 +360,30 @@ open class Kommunicate: NSObject,Localizable{
             }
         })
     }
+    
+    /**
+     Updates the conversation parameters.
+     Requires the conversation ID and the specific parameters that need to be updated for the specified conversation ID.
+
+     - Parameters:
+     - conversation: Conversation that needs to be updated
+     - completion: Called with the status of the conversation update
+     */
+    open class func updateConversation(conversation: KMConversation, completion:@escaping (Result<String, KommunicateError>) -> ()) {
+        
+        let service = KMConversationService()
+        if let groupID = conversation.clientConversationId, !groupID.isEmpty {
```

Comment:
We can change this `if` to `guard` because without the group ID we can't update anything.